datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearchHeader.tsx

```diff
@@ -94,12 +105,20 @@ export default function EmbeddedListSearchHeader({
                             entityRegistry={entityRegistry}
                             hideRecommendations
                         />
+                        {applyView && showSelectViewMode && (
```

Comment:
I'm not sure we need _both_ of the parameters - one should do, right? 

Basically if we pass applyView to the parent component, we should automatically give the ability to change the view

Reply:
So `applyView` is determining whether or not have the view be on the page, and `showSelectViewMode` is whether or not the view should be popped out of the hamburger menu thing. Does that change this at all or do you think I can do this in one?

Reply:
Ohhhh I think I misread something. Let me go again